### PR TITLE
Update : Blog Card With New Design

### PR DESCRIPTION
**Pull request by saadpasta**
Updated Blog Card With New Design that matches our theme. 

ezgif com-video-to-gif (1)

### CODE REVIEW

**Comment by kartik918** [comment]
Looks great! What's up with the Codacy error though? Says syntax incorrect, might want to look into that to keep the badge on README to A.

File: src/containers/blogs/Blog.css [hunk 1]
```diff
@@ -55,5 +55,14 @@
...
     line-height: normal;
     text-align: center;
+  }
+}
+
+@media screen and (max-width: 480px) and (min-width:300px){
+  .blog-text-div{
+    
+      grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
```
Comment (kartik918) [inline src/containers/blogs/Blog.css:65]: nitpick: reduce these empty lines

File: src/components/blogCard/BlogCard.css [hunk 2]
```diff
@@ -1,95 +1,140 @@
...
-  padding-left: 30px;
-  padding-right: 30px;
-  font-size: 12px;
-  color: #c8c8c8;
-  line-height: 18px;
-  overflow: hidden;
-  display: -webkit-box;
-  -webkit-line-clamp: 2;
-  -webkit-box-orient: vertical;
+
```
Comment (kartik918) [inline src/components/blogCard/BlogCard.css:92]: Remove extra line, I'll enforce this with prettier soon but for now we have to do this manually.

**Comment by saadpasta** [comment]
@kartik918 Done with changes and added prettier 🙂

**Pull request merged**